modules/flex/metadata.display.yaml

```diff
@@ -37,6 +37,9 @@ spec:
         container_spec_gcs_path:
           name: container_spec_gcs_path
           title: Container Spec Gcs Path
+          altDefaults:
+            - type: ALTERNATE_TYPE_DC
+              value: gs://dataflow-templates-us-central1/latest/flex/PubSub_to_BigQuery_Flex
```

Comment:
Curious as to why are we providing this default? Is this file publicly accessible? Does this default work for any type of dataflow job?

Reply:
Removed.